templates/env.tera

```diff
@@ -0,0 +1,8 @@
+---
+whiskers:
+  version: "2.3.0"
+  matrix:
+    - flavor
+  filename: "env/{{ flavor.identifier }}.env"
+---
+WHOOGLE_CONFIG_STYLE="{{- read_file(path="../css/" ~ flavor.identifier ~ ".css") | linebreaksbr | replace(from="<br>    ", to=" ") | replace(from="<br>}<br>", to=" }") | replace(from="<br>", to=" ") -}}"
```

Comment:
This looks and is cursed, but it is a limitation of the templating engine Whiskers uses that the `replace` filter cannot search for `\n` (newlines). Instead, we use another built-in filter called `linebreaksbr` that replaces newlines with the `<br>` element, and *then* we use the `replace` filter to replace the new `<br>` elements with spaces and such to minify it exactly as it was before (that's why there aren't any changes to the env files - this template perfectly matches it!).